resources/bower_components/angular/.bower.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please don't commit bower_components! Check that you use correct .gitignore file :)